dev/group_vars/is-analytics.yml

```diff
@@ -16,15 +16,18 @@
 
 wso2_group: wso2
 wso2_user: wso2carbon
-deb_setup_path: /usr/lib/wso2
-rpm_setup_path: /usr/lib64/wso2
 product_name: wso2is-analytics
 product_version: 5.8.0
-product_path: /opt/wso2
-product_archive_deb: wso2is-analytics-linux-installer-x64-5.8.0.deb
-product_archive_rpm: wso2is-analytics-linux-installer-x64-5.8.0.rpm
+product_path: /usr/local/wso2
 product_package_location: files
-jdbc_driver: mysql-connector-java-5.1.45-bin.jar
+jdbc_driver: mysql-connector-java-5.1.47-bin.jar
+
```

Comment:
Shall we remove the extra line here?

Reply:
This was done to separate JDK related variables from the rest. Shall I add a comment indicating that the variables after the space are related to JDK distributions?

Reply:
Fixed https://github.com/wso2/ansible-is/pull/17/commits/b99772a015e08081ff91fd679fec49857ae63e14